src/api/entity_alias/responses.rs

```diff
@@ -0,0 +1,16 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Deserialize, Debug, Serialize)]
+pub struct CreateEntityAliasResponse {
+    pub request_id: String,
+    pub lease_id: String,
+    pub renewable: bool,
+    pub lease_duration: i64,
```

Comment:
`lease_dorations` are always positive seconds, so we should use `u64` 
```suggestion
    pub lease_duration: u64,
```